api2-war/src/main/java/eu/europeana/api2/v2/web/controller/SearchController.java

```diff
@@ -127,43 +138,46 @@
     @Value("${api.search.hl.MaxAnalyzedChars}")
     private String hlMaxAnalyzedChars;
 
+    public static void main(String args[]) {
+        String[] profiles = {"amit", "rahul", "surya"};
+        System.out.println(StringUtils.join(profiles, ","));
+    }
+
```

Comment:
Please delete testing code, or change it into a proper unit test

Reply:
I don't know where that method comes from: it suddenly appears on dec 15 on my local history overview, but I certainly have not consciously added it. I don't even understand what the point of it is.

Reply:
I'll take it out.